docs/DEPLOY.md

```diff
@@ -0,0 +1,19 @@
+# Deploy
+
+##### `sls deploy`
+- Description: Deploy resource group containing function app and other resources. Also deploys Function App code
+- Two Approaches:
+  - Deploy resource group, upload packaged artifact directly to function app. Sets function app `RUN_FROM_PACKAGE` setting to `1`.
+  - Deploy resource group, upload packaged function app to blob storage with version name. Sets function app `RUN_FROM_PACKAGE` setting to path of zipped artifact in blob storage
+- Open Questions:
+  - Default to deploy to blob storage? Default to zip deploy directly to function app?
```

Comment:
My vote is on zip deploy default to function app and not blob storage.  This default will be more useful for the general user so they don't incur extra cost maintaining a bloc storage.

Reply:
Sounds good. I think that would just mean that `rollback` would not be an option. Something to be aware of is that a function app basically requires a blob storage to be deployed alongside it